README.md

```diff
@@ -1 +1,30 @@
 # apertium-incubator
+
+[Apertium][1] is a free/open-source platform for developing rule-based machine
+translation systems.
+
+This repository holds submodules to Apertium languages, pairs, and other
+modules, that are classified as [incubator][2] modules.
```

Comment:
@sushain97 This is the only thing that will be different for each tier-1 repo, just confirming the sentence is alright.

Reply:
Hm, word "modules" is used twice in different senses. One can probably be switched to packages? Packages is the more general term whereas I've seen module used to describe e.g. apertium-hin.

Reply:
How about "that are in the Apertium incubator"?

Reply:
sounds good!